subsys/bluetooth/services/Kconfig.nus

```diff
@@ -11,6 +11,11 @@ config BT_NUS
 	  Enable Nordic UART service.
 if BT_NUS
 
+config BT_NUS_AUTHEN
+	bool "Require authentication for Nordic UART service"
```

Comment:
It is nice to keep prompt short. Maybe just "Require authentication"

Reply:
@KAGA164 Okay, I spent a little time looking around and deleted the help text, while using the prompt [from this option](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/kconfig/index.html#CONFIG_BT_HRS_DEFAULT_PERM_RW_AUTHEN). It best describes what it does and makes it consistent in the tree. Do you agree?

Reply:
The prompt test is good now, but please bring back the help text to render this nice in the documentation. If you want to do it in consistent way with other services you can use choice in Kconfig like it is done in the HRS https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/services/Kconfig.hrs#L11
https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/services/hrs.c#L46.
If you decide to go with approach from HRS, you can also add option for access with encryption

Reply:
@KAGA164 Okay. Let's keep it simple for now - I have returned back the help text. But we can consider the choice option in the future. The HRS sample gives the most flexibility.